styles/_theme.scss

```diff
@@ -659,3 +397,26 @@ blockquote .source {
   padding: 0;
   list-style-position: inside;
 }
+
+.usa-prose {
+  .usa-collection__item {
+    border-top: none;
+  }
+}
+
+h1 {
+  font-family: Source Sans Pro Web, Helvetica Neue, Helvetica, Roboto, Arial, sans-serif;
```

Comment:
as mentioned above, use USWDS utility conventions (like sass mixins with the design tokens) here and in line 413 below